docs/source/traitsui_user_manual/factories_advanced_extra.rst

```diff
@@ -1285,6 +1285,30 @@ For example, the default editor for Range(low=0, high=1500) has
 
     View(Item('my_range', editor=DefaultOverride(high_label='Max'))
 
+VideoEditor()
+`````````````
+
+VideoEditor() is a display-only video editor. Note that this editor is only
+available on Qt at the moment. Please see :ref:`TraitsUI Demos <traitsui-demo>`
+for a demo.
+
+.. note:: Depending on your operating system, you might have to install
+    external codecs to get the VideoEditor working.
+
+    On Windows, you will need to install the
+    `K-Lite Codec Pack<https://codecguide.com/about_kl.htm>` if you are using EDM
+    to install Qt5. This is because the EDM build of Qt5 is built to use
```

Comment:
Suggested rephrasing:

> On Windows, you will need to install DirectShow filters (such as those available in the K-Lite Codec Pack) in order to play most video formats when using EDM to install Qt5. This is because...

Reply:
resolved.